WORKSPACE

```diff
@@ -59,7 +60,6 @@ git_repository(
 
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kotlin_repositories")
 
-KOTLIN_VERSION = "1.3.31"
 
 KOTLINC_RELEASE_SHA = "107325d56315af4f59ff28db6837d03c2660088e3efeb7d4e41f3e01bb848d6a"
```

Comment:
I don't mind the idea of the versions file, but I think the SHAs should go there too. When you update one you'll need to update the other won't you?

Reply:
Good call. I'll move the SHAs.